WindowState/README.md

```diff
@@ -0,0 +1,127 @@
+# WindowState - Surface Duo Compose SDK
+
+**WindowState** is a component for Jetpack Compose, help you easily get details about the window state of the dual-screen, foldable and large screen devices, using the Google [Jetpack WindowManager](https://developer.android.com/jetpack/androidx/releases/window) library.
+
+The component provides the current window information as a Compose state, including folding position, orientation and window size classes. For the dual-screen and foldable device, combining different folding position and orientation, we introduce four display postures to take advantage of these new form factors, Dual Portrait, Dual Landscape, Single Portrait, Single Landscape. 
+
+![postures](screenshots/postures-overview.png)
+
+And the window size classes are measured based on Google [Window size classes](https://developer.android.com/guide/topics/large-screens/support-different-screen-sizes) to help you support different screen sizes of the devices, including large screen and regular single screen devices.
```

Comment:
How about changing the link to this so it goes directly to the window size class header: https://developer.android.com/guide/topics/large-screens/support-different-screen-sizes#window_size_classes